vita3k/config/include/config/config.h

```diff
@@ -95,6 +95,7 @@ enum PerfomanceOverleyPosition {
     code(bool, "discord-rich-presence", true, discord_rich_presence)                                    \
     code(bool, "wait-for-debugger", false, wait_for_debugger)                                           \
     code(bool, "color-surface-debug", false, color_surface_debug)                                       \
+    code(bool, "show-touchpad-cursor", true, show_touchpad_cursor)                                      \
```

Comment:
On a PSTV, is a cursor begin displayed when using the DS4 touchpad?

Reply:
yes, pstv display cursor, but my cursor is custom